### PR TITLE
LPS-181405 | master

### PR DESCRIPTION
**Pull request by Tim-Cao**
https://issues.liferay.com/browse/LPS-181405

This is only for master.

@david-gutierrez-mesa

The reason to allocate smaller cases to dedicated jenkins nodes is ci will group upgrade cases with same dump without rebuild database. That will cause them failed by wrong data.

**Before break down**, the ci duration info is
```
Build Time: 2 hours 5 minutes

Total CPU Usage Time: 2 days 2 hours 53 minutes

Total number of Jenkins slaves used: 144
```

**After break down**, the ci duration info is
```
Build Time: 2 hours 22 minutes

Total CPU Usage Time: 2 days 1 hour 55 minutes

Total number of Jenkins slaves used: 160
```

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/webcontent/WebContentUpgrade.testcase [hunk 1]
```diff
@@ -918,149 +918,213 @@ definition {
...
 		property portal.version = "7.0.10.14";
-
-		Upgrade.ViewWebContentWCStructureWCTemplateWCDandAssetPublisher(webContentColor = "false");
+		property test.name.skip.portal.instance = "WebContentUpgrade#EditWebContentAndViewInWCDAndAssetPublisher701014";
 
 		Upgrade.EditWebContentAndViewInWCDandAssetPublisher();
+	}
 
-		Upgrade.EditWCStructureAndTemplateAndViewInWCDandAssetPublisher(webContentColor = "false");
+	@priority = 5
```
Comment (david-gutierrez-mesa) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/webcontent/WebContentUpgrade.testcase:1097]: Missing description

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/webcontent/WebContentUpgrade.testcase [hunk 2]
```diff
@@ -918,149 +918,213 @@ definition {
...
+		property data.archive.type = "data-archive-web-content-draft";
+		property database.types = "mariadb,mysql,postgresql";
+		property portal.version = "6.2.5";
+
+		Upgrade.addWCStructureAndTemplate();
+
+		Upgrade.publishWCDraft();
+	}
+
+	@priority = 5
```
Comment (david-gutierrez-mesa) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/webcontent/WebContentUpgrade.testcase:1108]: Missing description

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/webcontent/WebContentUpgrade.testcase [hunk 3]
```diff
@@ -918,149 +918,213 @@ definition {
...
+		property data.archive.type = "data-archive-web-content-draft";
+		property database.types = "db2,mariadb,mysql,oracle,postgresql,sybase";
+		property portal.version = "6.2.10.15";
+
+		Upgrade.addWCStructureAndTemplate();
+
+		Upgrade.publishWCDraft();
+	}
+
+	@priority = 5
```
Comment (david-gutierrez-mesa) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/webcontent/WebContentUpgrade.testcase:1119]: Missing description

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/webcontent/WebContentUpgrade.testcase [hunk 4]
```diff
@@ -1083,6 +1147,94 @@ definition {
...
+	test ViewWebContentWCStructureWCTemplateWCDAndAssetPublisher721 {
+		property data.archive.type = "data-archive-web-content-with-fields";
+		property database.types = "mariadb,mysql,postgresql";
+		property portal.version = "7.2.1";
+		property test.name.skip.portal.instance = "WebContentUpgrade#ViewWebContentWCStructureWCTemplateWCDAndAssetPublisher721";
+
+		Upgrade.ViewWebContentWCStructureWCTemplateWCDandAssetPublisher();
+	}
+
+	@description = "This is a test for LPS-126656. Upgrade 7.3 to Master with a Web Content, a Structure, a WC Template and a page with Web Content Display and Asset Publisher"
```
Comment (david-gutierrez-mesa) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/webcontent/WebContentUpgrade.testcase:1183]: We have 6 descriptions that are exactly the same. Maybe we can say here ".. Upgrade 7.3.5 to Master w..." Could we take advantage of this PR and update all the descriptions?

**Review by david-gutierrez-mesa — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/webcontent/WebContentUpgrade.testcase (hunk 1 above)
Comment (Tim-Cao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/webcontent/WebContentUpgrade.testcase:1097]: The missing one is out of this refactor. It is an another test.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/webcontent/WebContentUpgrade.testcase (hunk 2 above)
Comment (Tim-Cao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/webcontent/WebContentUpgrade.testcase:1108]: The missing one is out of this refactor. It is an another test.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/webcontent/WebContentUpgrade.testcase (hunk 3 above)
Comment (Tim-Cao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/webcontent/WebContentUpgrade.testcase:1119]: The missing one is out of this refactor. It is an another test.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/webcontent/WebContentUpgrade.testcase (hunk 4 above)
Comment (Tim-Cao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/webcontent/WebContentUpgrade.testcase]: Thanks. Added them here https://github.com/liferay-echo/liferay-portal/pull/11731/commits/fd6c2a63fb96cfbb3ea8f9c3a1a53397d7477b74

**Comment by Tim-Cao** [comment]
ci:forward

**Pull request closed without merging**